src/components/grid-aware/StatsBarBlock/StatsBarBlock.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from "react";
+import { ThemeColorOption, TitleColorOption } from "../../../types";
+
+import * as s from "./StatsBarBlock.module.css";
+
+const { Fragment } = React;
+
+type BarProps = {
+  label: string | number;
+  statistic: string | number;
+};
+
+type BarStatProps = {
+  bars: BarProps[];
+  title: string;
+  subtitle: string;
+  orientation: "vertical" | "horizontal";
+};
+
+const BarStatCard = ({ bars, title, subtitle, orientation }: BarStatProps) => (
+  <div className={s.card}>
+    <div className={`${s.barsContainer} ${s[orientation]}`}>
+      {bars.map(({ label, statistic }, i) => {
+        let barSize;
+
+        switch (i) {
+          case 0:
+            barSize = "small";
+            break;
+          case 1:
+            barSize = "medium";
+            break;
+          case 2:
+            barSize = "large";
+            break;
+          default:
+            barSize = "large";
+        }
+
+        return (
+          <Fragment key={`${label} ${statistic}`}>
+            <div className={`${s.label} ${s[orientation]}`}>{label}</div>
+            <div className={`${s.barStatisticWrapper} ${s[orientation]}`}>
+              <div className={`${s.bar} ${s[barSize]} ${s[orientation]}`} />
+              <div className={`${s.statistic} ${s[orientation]}`}>
+                {statistic}
+              </div>
+            </div>
+          </Fragment>
+        );
+      })}
+    </div>
+    <div className={`${s.cardTitle} ${s[orientation]}`}>{title}</div>
+    <div className={s.cardSubtitle}>{subtitle}</div>
+  </div>
+);
+
+type StatsBarBlockProps = {
+  title: string;
+  subtitle?: React.ReactNode;
+  barStatCards: BarStatProps[];
+  theme: ThemeColorOption;
+  titleColor?: TitleColorOption;
+};
+
+const StatsBlock = ({
+  title,
+  subtitle,
+  barStatCards,
+  theme,
+  titleColor = "black",
+}: StatsBarBlockProps) => (
+  <div
+    className={`${s.bleedWrapper} 
+      ${s[theme]}`}
+  >
+    <div
+      className={`${s.bleedBackground} 
+        ${s[theme]}`}
+    />
```

Comment:
(Sorry, forgot to actually attach this to my review)

Since these will all fit on a single line, could you make it so that each of these `<div>`s take up only a single line? I think it'll be more readable that way.

```suggestion
  <div className={`${s.bleedWrapper} ${s[theme]}`}>
    <div className={`${s.bleedBackground} ${s[theme]}`}/>
```